ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -642,32 +1000,69 @@ def _resolveNewGeneration(self, traj, rlz, objectiveVal, fitness, g, info):
       @ In, g, xr.DataArray, the constraint evaluation function
       @ In, info, dict, identifying information about the realization
     """
-    self.raiseADebug('*'*80)
-    self.raiseADebug(f'Trajectory {traj} iteration {info["step"]} resolving new state ...')
+    self.raiseADebug(f'Trajectory {traj} iteration {info["step"]} resolving new Generation (population) ...')
     # note the collection of the opt point
     self._stepTracker[traj]['opt'] = (rlz, info)
     acceptable = 'accepted' if self.counter > 1 else 'first'
     old = self.population
     converged = self._updateConvergence(traj, rlz, old, acceptable)
     if converged:
-      self._closeTrajectory(traj, 'converge', 'converged', self.bestObjective)
+      self._closeTrajectory(traj, 'converge', 'converged', self.multiBestObjective if self._isMultiObjective else self.bestObjective)
     # NOTE: the solution export needs to be updated BEFORE we run rejectOptPoint or extend the opt
     #       point history.
+
+    if self._isMultiObjective:
+      objVal = xr.DataArray(np.array(self.objectiveVal).T, #objVal,
+                            dims=['chromosome','obj'],
+                            coords={'chromosome':np.arange(len(self.objectiveVal[0])),
+                                    'obj': self._objectiveVar})
```

Comment:
@joshua-cogliati-inl, I think objVal is no longer used. Maybe this if statement should be removed.

Reply:
I agree, this if statement should be removable.

Reply:
https://github.com/joshua-cogliati-inl/raven/commit/84a75705c98d3f9044017dc1cb73b00dac698e4b